src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -202,7 +202,14 @@ ClientRequest::process_op(instance_handle_t &ihref, Ref<PG> &pg)
   ).then_interruptible(
     [this, pg]() mutable {
     if (pg->is_primary()) {
-      return do_recover_missing(pg, m->get_hobj());
+      auto fut = interruptor::now();
+      auto soid = m->get_hobj();
+      if (!soid.is_head()) {
+        fut = do_recover_missing(pg, soid.get_head());
+      }
+      return fut.then_interruptible([this, pg]() mutable {
+        return do_recover_missing(pg, m->get_hobj());
+      });
```

Comment:
could you please to introduce which scenario need recovery head first for clone object?